railties/test/generators/shared_generator_tests.rb

```diff
@@ -28,9 +28,9 @@ def test_skeleton_is_created
 
   def assert_generates_with_bundler(options = {})
     generator([destination_root], options)
-    generator.expects(:bundle_command).with('install').once
-    generator.stubs(:bundle_command).with('exec spring binstub --all')
```

Comment:
We don't have to use the method call assertions. If we use a `Minitest::Mock` directly can't we mock `install` and stub `exec spring binstub --all`?